test/testdata/infer/lazy_is_a_false.rb

```diff
@@ -0,0 +1,4 @@
+# typed: false
+
+T::LazyConstants.lazy_is_a?('::Integer') # error: in `# typed: true` files
+T::LazyConstants.lazy_is_a?('::DoesntExist') # error: in `# typed: true` files
```

Comment:
Do these need two arguments?

Reply:
The number of arguments doesn't matter for that error, because it's coming when resolver sees a method call matching `T::LazyConstants.lazy_is_a?` syntactically (it's designed to tell people that it won't work right in `typed: false` files).